backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -464,6 +470,80 @@ async function getPublicKey(email: string): Promise<string | null> {
   return loginUser.pubKey.toString('hex')
 }
 
+const protoTransaction = (
```

Comment:
Why do we have this function twice? What is the difference to the one in the AdminResolver?

Reply:
You are perfectly right - this still has to be unified. The main difference is what is signed into the tx hash as I understand it. In every manual transaction we have `sodium.crypto_generichash_update(state, 'start decay')` and in those generated by the system for the transactions and creations we don't update the generic crypto hash with a string value like that. But I am not 100% sure yet. I implemented this in two different versions for now since it is implemented twice (or more times) in the login_server and some more implementations on the community_server. I expect those processes to varry alot and to have to be adapted based on the findings when testing for 100% value reproduction. Later on I will ofc unify those.